Docs 6867 visualizer visualization doc move (#59)
mulesoft/docs-visualizer: modules/ROOT/pages/architecture-visualization.adoc

> = Architecture Visualization

Reviewer: HERE

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

> == About the Architecture Visualization

Reviewer: GLOBALLY for this documentation, omit "the" before "<Name> Visualization," or follow the name with a noun object, like "feature": the Architecture Visualization feature.
I will stop marking this, so please use search.

> The Architecture Visualization shows the topology of your application network. In this visualization, you can change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.

Author: ```suggestion
Architecture Visualization shows the topology of your application network. In this visualization, you can change display names of nodes, add tags, and assign nodes to layers. Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
```

> The Architecture Visualization shows the topology of your application network. In this visualization, you can assign change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.

Reviewer 2: In this visualization, you can change display names .... " There is an extra word (assign) in this sentence

Author: ```suggestion
The Architecture Visualization shows the topology of your application network. In this visualization, you can change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.
```

> The Architecture Visualization shows the topology of your application network. In this visualization, you can assign change display names of nodes, add tags, and assign nodes to layers. The Architecture Visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.

Reviewer: "...you can assign change display names..."? Do mean "you can assign or change display names"?

Reviewer: Is this view most useful for doc/preso because you can share without exposing metrics, or do we mean that it is most useful when you do not want to expose metrics, such as in doc or preso?

> == View Information about One Service in your Network

Reviewer: ```suggestion
== View Information About One Service in Your Network
```
https://capitalizemytitle.com/

> When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.

Reviewer: Does your audience know what graph we are referring to?

Author: @asimons-mule should we say "canvas" here instead?

Reviewer 2: Yes, canvas will work

Anypoint Visualizer displays information about the node in a card. This card is divided into the following tabs:

> * *Details*: Provides information about the node.

Reviewer: GLOBALLY limit list items (and steps) to one item, fragment, or sentence, per our style ref.
Avoid using this single-line colon format except with very brief texts. This first one is OK, but not the next one.
The point is SCANNABILITY (and parallelism within the list), so...:
```suggestion
* *Details*
Provides information about the node.
```

> * *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc[tags] from this section.

Reviewer 2: The Layers link goes to a specific section (setting via UI) and the tags link goes to the entire tags page. Let's make this consistent. I don't have a preference either way.

Author: ```suggestion
* *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc#set-tag-ui[tags] from this section.
```

> * *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc[tags] from this section.

Reviewer: ```suggestion
* *Customization*
Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc[tags] from this section.
```

> * *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc#set-tag-ui[tags] from this section.

Author: ```suggestion
* *Customization* 
Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc#set-tag-ui[tags] from this section.
```

> * *Customization*: Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc#set-tag-ui[tags] from this section.

Author: ```suggestion
* *Customization*
Provides information about the layer to which the node is assigned and the tags assigned to the node. You can xref:customize-your-canvas.adoc#display-name[edit the node's display name] or xref:layers.adoc#set-a-layer-ui[change the layer] and xref:use-tags-in-visualizer.adoc#set-tag-ui[tags] from this section.
```

> == Filtering your View

Reviewer: ```suggestion
== Filtering Your View
```

Reviewer: Just an idea:

Consider a different intro here: You can search for nodes by using.... Then you can select them, with or without their dependents, or you can hide them from the canvas.

...followed by tasks for Search, Select, Hide....?

> When you select a node, you can click *Select dependents and dependencies*, which selects all nodes related to the service you selected.

Reviewer: ```suggestion
When you select a node (service) that has dependents and dependencies, you have the option to also select all nodes related to the service you selected. If the node you select does not have dependents and dependencies, this option is not offered.
```
Do we have to equate node with service? If possible, suggest using just one term throughout.
I moved this parenthetical up from where it appears below.

> If a single node is selected and does not have any dependents or dependencies, this option is hidden.

Reviewer: OMIT per above.

> If a single node is selected and does not have any dependents or dependencies, this option is hidden.

Author: ```suggestion
```

> If a single node is selected and does not have any dependents or dependencies, this option is hidden.

Author: ```suggestion
```

> You can hide nodes through what you select in the canvas.

Reviewer: ```suggestion
You can hide nodes through what you select in the canvas:
```

* To hide a single node, select it, and then click *Hide selected service*.
* To hide multiple nodes, multiselect them, and then click *Hide selected services*.

> * To hide all of the services other than the selected nodes, you can click *Hide other services*. If you want to hide everything that is not related to a given dependency stack, select a node and then click *Select dependents and dependencies*, which selects all nodes related to the service which you selected. Then, click *Hide other services*.

Reviewer: Is all of the text in the third bullet item about hiding all but selected nodes?
If so, break after first sentence.
If not, create a separate list item.
This list is a little tricky because it looks like it is supposed to list alternative single-step options, but the options are mini-tasks: Hide Single Node, Hide Multiple Nodes, Hide All But Selected Nodes....

Author: ```suggestion
* To hide all of the services other than the selected nodes, you can click *Hide other services*. 
If you want to hide everything that is not related to a given dependency stack, select a node and then click *Select dependents and dependencies*, which selects all nodes related to the service which you selected. Then, click *Hide other services*.
```

> You can search for a node (service) in the *Search services* text field. Services containing your search in any part of their name are highlighted, while others fade.

Reviewer: ```suggestion
You can search for a node in the *Search services* text field. Services containing your search in any part of their name are highlighted, while others fade.
```
Should searching come before the other actions in this subtopic?

Author: Filtering is the happier path than searching, I think.

[[export-view]]
== Export Your Network View

> You can export the view of your network showing on your canvas to a PNG file.

Author: ```suggestion
You can export the view of your network showing on your canvas to a PNG file by clicking the *Export as Image* button next to the zoom controls in the bottom right.
```

If you customized your view, the customized network topology is exported.

> To export a view of your network as a PNG, click the *Export as Image* button next to the zoom controls in the bottom right.

Reviewer: Seems like we could replace all of this with one sentence:
To export your network view to a PNG file, click *Export as Image*.
IF this isn't obvious, we could add "If you customized the view, that is the view exported." 
Take a look at your sentences and evaluate them for verbosity. We don't need to name and rename things after a context is established, and we should assume a certain level of user sophistication (if *I* can get it, anyone can!).

> To export a view of your network as a PNG, click the *Export as Image* button next to the zoom controls in the bottom right.

Author: ```suggestion
```

= Architecture Visualization
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

== About the Architecture Visualization

The Architecture visualization shows the topology of your application network. In this visualization, you can change display names of nodes, add tags, and assign nodes to layers. The Architecture visualization is most useful for documentation and presentation purposes since you can share your network’s architecture without visible metrics.

== View Information About One Service in Your Network

> When you select a node within the graph, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.

Author: ```suggestion
When you select a node within the canvas, the node and its dependency stack are highlighted and all unconnected nodes are unavailable.
```
